release-please-config.json

```diff
@@ -1,18 +1,13 @@
 {
-  "bootstrap-sha": "505795a1312e90f1ce3b59b530622929bc38b4fe",
+  "last-release-sha": "48545ebe503d661af0562b240648253199758d9f",
```

Comment:
Is this necessary? I believe we'd need to remove it after the next successful release, so if it isn't necessary it might be better not to add it.

Reply:
It's required when there's a change in the tag format, but since we're not doing that, this is unnecessary.